align/compiler/find_constraint.py

```diff
@@ -295,6 +295,9 @@ def FindConst(graph, name, ports, ports_weight, input_const, stop_points=None):
                     pins2=s2
                 ))
                 continue
+        elif isinstance(const, constraint.DoNotIdentify):
+            written_symmetries +=','.join(const.instances)
```

Comment:
Does the check on line 329 fail if key or value is at the end of written_symmetries? Example: written_symmetries = 'net1,net2,**net3**'

How about making written_symmetries a set rather than string to prevent above?